cmake/NeuronFileLists.cmake

```diff
@@ -583,7 +588,7 @@ nrn_create_file_list(NRN_NRNMPI_SRC_FILES ${PROJECT_SOURCE_DIR}/src/nrnmpi ${NRN
 nrn_create_file_list(NRN_NRNGNU_SRC_FILES ${PROJECT_SOURCE_DIR}/src/gnu ${NRNGNU_FILES_LIST})
 nrn_create_file_list(NRN_NRNPYTHON_SRC_FILES ${PROJECT_SOURCE_DIR}/src/nrnpython
                      ${NRNPYTHON_FILES_LIST})
-nrn_create_file_list(NRN_MODFILE_BASE_NAMES ${PROJECT_SOURCE_DIR}/src/nrnoc ${MODFILE_BASE_NAMES})
+nrn_create_file_list(NRN_MODFILE_BASE_NAMES src/nrnoc ${MODFILE_BASE_NAMES})
```

Comment:
Why do we need this change? It makes it inconsistent with the surrounding lines, and I am not sure of the benefit.

Reply:
Using paths relative to PROJECT_SOURCE_DIR makes the macro `nocmodl_mod_to_c` (defined [here](https://github.com/neuronsimulator/nrn/pull/1673/files#diff-992bc6e6741c7dad01728000dc368ce09ae872ce52d4968d4c6af26ca2d94abcR239-R248) and used [here](https://github.com/neuronsimulator/nrn/pull/1673/files#diff-8d4f4d49fd7914f9310f8cdaa55667ec744ea98cf53752f52940d942ca3559c5R85-R91) a bit simpler than if they remain absolute paths. This is the only location where this variable is used.

Besides, IMO this change is more consistent in regards to _BASE_NAMES_ in `NRN_MODFILE_BASE_NAMES` variable.